mtp_noms_ops/apps/security/utils.py

```diff
@@ -173,6 +173,11 @@ def is_hmpps_employee(user):
     return hmpps_employee_flag in flags
 
 
+def has_provided_job_information(user):
+    flags = user.user_data.get('flags') or []
```

Comment:
i like this more than `user.user_data.get('flags', [])`

Reply:
Cool, I find it more readable but consistency is as important, to keep it consistent with the use of default arguments only a few lines below (line 182 of this file), I actually think it might be better using the `default` argument and refactor the other uses in one go if you do like the more verbose syntax better?

What do you think?

Reply:
i'm 110% for consistency. so the pedant in me would change other occurrences to `user.user_data.get('flags') or []`